signer-wasm/src/lib.rs

```diff
@@ -168,6 +168,7 @@ pub fn key_recover(private_key_js: JsValue, testnet: bool) -> Result<ExtendedKey
 pub fn transaction_serialize(unsigned_message: JsValue) -> Result<String, JsValue> {
     set_panic_hook();
 
+    // TODO: rename; can be unsigned or signed message (so maybe 'message')
```

Comment:
Yes, makes sense. Let's better rename it now so we don't forget.

Reply:
I actually noticed another issue related to that https://github.com/Zondax/filecoin-rs/issues/174